### PR TITLE
[SDK-3910] Flutter quickstart amendments

### PR DESCRIPTION
**Pull request by stevehobbsdev**
- fix: use inline values for domain and scheme instead of strings.xml
- fix: be explicit about which 'build.gradle' file to edit

### CODE REVIEW

File: articles/quickstart/native/flutter/01-login.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This quickstart assumes you already have a [Flutter](https://flutter.dev/) appli
 
 You should also be familiar with the [Flutter command line tool](https://docs.flutter.dev/reference/flutter-cli).
 
-<%= include('_configure_urls_interactive') %>
+<%= include('\_configure_urls_interactive') %>
```
Comment (Widcket) [inline articles/quickstart/native/flutter/01-login.md]: Is `\` correct here?

File: articles/quickstart/native/flutter/interactive.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ This quickstart assumes you already have a [Flutter](https://flutter.dev/) appli
 
 You should also be familiar with the [Flutter command line tool](https://docs.flutter.dev/reference/flutter-cli).
 
-<%= include('_configure_urls_interactive') %>
+<%= include('\_configure_urls_interactive') %>
```
Comment (Widcket) [inline articles/quickstart/native/flutter/interactive.md]: Same here.

File: articles/quickstart/native/flutter/01-login.md (hunk 1 above)
Comment (stevehobbsdev) [reply articles/quickstart/native/flutter/01-login.md]: No, for some reason VSCode inserts a slash when auto-formatting and I'd forgotten to turn it off. Removed it now, thanks

File: articles/quickstart/native/flutter/interactive.md (hunk 2 above)
Comment (stevehobbsdev) [reply articles/quickstart/native/flutter/interactive.md]: Thanks, removed in [045199e](https://github.com/auth0/docs/pull/10202/commits/045199e8267c57c80e1e727a6415a91e843022e1)

**Review by Widcket — APPROVED** [review]

**Pull request merged**